src/utils/CatalystClientBuilder.ts

```diff
@@ -0,0 +1,148 @@
+import { getMainnetCatalysts, getRopstenCatalysts } from 'dcl-catalyst-commons'
+import { CatalystClient } from '../CatalystClient'
+import CatalystsList from '../CatalystsList'
+
+/**
+ * Returns a CatalystClient connected to one of the catalysts in the given network
+ */
+export async function clientConnectedToCatalystIn(
+  network: 'mainnet' | 'ropsten',
+  origin: string
+): Promise<CatalystClient> {
+  const noContractList = await getUpdatedApprovedListWithoutQueryingContract({
+    preKnownServers: { network },
+    origin
+  })
+  let list: string[]
+  if (noContractList) {
+    list = noContractList
+  } else {
+    console.warn('Falling back to the smart contract')
```

Comment:
Maybe we should include what we are doing here? Something like `Falling back to the smart contract to get an updated list of active servers`?

Reply:
Done!